app/controllers/HomePageController.scala

```diff
@@ -56,18 +56,14 @@ class HomePageController @Inject() (
   }
 
   private def getDownloadLinkMessagesKey(opt: Option[DownloadDataSummary], doesGoodsRecordExist: Boolean): String =
-    if (doesGoodsRecordExist) {
+    if (!doesGoodsRecordExist) {
```

Comment:
having ! before a check in an if is less readable - could you switch it back to the old order?